apache-activemq-mixin/dashboards/dashboards.libsonnet

```diff
@@ -0,0 +1,5 @@
+(import 'apache-activemq-cluster-overview.libsonnet') +
```

Comment:
Up to you if you wanna do this now or wait and see, but based off the feedback i recently got from Vitaly you're probably gonna need to add "/ $__interval" to the end of all your panels that use increase

Reply:
Going to wait for a bit. I might decide to change into rate, I want to see how it looks under sample app first.